### PR TITLE
Add meaningful page titles for reports

### PR DESCRIPTION
**Pull request by rsanheim**
Adds page titles to the report pages, as well as making the env name a shorter, more clear abbreviation for better tab readability.

### CODE REVIEW

File: app/helpers/simple_server_env_helper.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 module SimpleServerEnvHelper
-  CUSTOMIZED_ENVS = %w[development qa sandbox demo production].freeze
+  CUSTOMIZED_ENVS = %w[development demo qa sandbox production].freeze
```
Comment (harimohanraj89) [inline app/helpers/simple_server_env_helper.rb:2]: How far reaching is this little constant? I'm wondering if we can easily add real support for our review envs as well while we're here, or whether that's a task for later.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -17,7 +17,7 @@
       </script>
     <% end %>
     <title>
-      <%= get_title_for_environment %>
+      <%= page_title %>
```
Comment (harimohanraj89) [inline app/views/layouts/application.html.erb:20]: Ah interesting. We just said `Simple Dashboard` everywhere, and now this method lets us define additional content per-page. Toit.

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -2,6 +2,12 @@ module ApplicationHelper
   DEFAULT_PROGRAM_INCEPTION_DATE = Time.new(2018, 0o1, 0o1)
   STANDARD_DATE_DISPLAY_FORMAT = "%d-%^b-%Y"
 
+  def page_title
+    str = get_title_for_environment
+    str << " |" << content_for(:title) if content_for?(:title)
```
Comment (harimohanraj89) [inline app/helpers/application_helper.rb:7]: I wonder if we can do `[ENV] <content> | Simple Dashboard`, or even remove `Simple Dashboard` from the titles? I'm looking at Github page titles for reference, and they don't mention "Github" in the page titles - I guess we let the icons do the talking.

**Review by harimohanraj89 — APPROVED** [review]
Code looks solid. Just content suggestions from me.

We can ship this piecemeal if needed.

File: app/helpers/application_helper.rb (hunk 3 above)
Comment (rsanheim) [reply app/helpers/application_helper.rb:7]: Yup, I'll add that for when there is an explicit title provided.

**Review by kpethtel — APPROVED** [review]
Nicely done.

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (rsanheim) [reply app/views/layouts/application.html.erb:20]: 👍

File: app/helpers/simple_server_env_helper.rb (hunk 1 above)
Comment (rsanheim) [reply app/helpers/simple_server_env_helper.rb:2]: Yeah we can always add that later, I'm guessing review falls into the `default` bucket for now.

**Pull request merged**